src/components/InputNumber.js

```diff
@@ -81,11 +76,11 @@ const InputNumber = React.forwardRef(
 
         let newValue = value;
         if (isDeleteAll) {
-          newValue = 0;
+          newValue = 0n;
         } else if (isNumberChar) {
-          newValue = value * 10 + Number(evt.key);
+          newValue = value * 10n + BigInt(evt.key);
         } else if (isBackspace) {
-          newValue = Math.floor(value / 10);
+          newValue = value / 10n;
```

Comment:
question: Are all division with bigint an integer division?

Just to make sure we dont introduce a float here out of nowhere.